content/docs/minion-v0.4/tag_cache.md

```diff
@@ -16,10 +16,8 @@ showdisqus = true
 ## Motivation
 
 The tagged memory mechanism augments each data word in memory with a small
-piece of extra metadata, a _tag_.
-To store these tags in generic memories such as DDR3/4,
-a small section in the memory space (normally near the upper boundary)
-is reserved for tags and hidden from applications.
+piece of extra metadata, a _tag_. Due to the mismatch in width between tag cache and mobile DDR,
```

Comment:
Is "mobile DDR" the right term?
I am not sure what is the "mismatch in width between tag cache and mobile DDR".